nixos/modules/services/x11/xidlehook.nix

```diff
@@ -0,0 +1,225 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.xserver.xidlehook;
+in {
+  options = {
+    services.xserver.xidlehook = {
+      enable = mkEnableOption "xidlehook";
+      timers = mkOption {
+        description = ''
+          List of timers.
+
+          Most other xidlehook options adjust the default value of this option;
+          if this option is set directly, most other xidlehook options have no effect.
+          This is done to maintain drop-in compatibility with the xautolock module
+          for folks that already have xautolock set up the way they like it.
+          If you don't have an existing xautolock config, you may prefer the
+          simplicity of xidlehook's one-list-of-timers interface.
+       '';
+        example = [
+          # This is upstream's example: Dim the screen after 60 seconds, undim if user becomes active
+          {
```

Comment:
You need to wrap this whole example in `literalExample '' ... ''` for it to show up nicely in the manual